### ISSUE TITLE
Run animations in screen shot mode

### ISSUE THREAD

**Issue by JanEricNitschke** [HSF/phoenix#634]
Is there currently a way to run the preset animations while in screen shot mode? This would allow for nicely recording them using screen capture software such a obs.

**Comment by EdwardMoyse** [comment]
No, not yet. I actually wanted to support this via a batch mode, which I think is doable...
